File: server/core/ViewModels/LibraryEntryViewModel.cs
namespace Wbs.Core.ViewModels;

public class LibraryEntryViewModel
{
    public string OwnerId { get; set; }
    public string EntryId { get; set; }
    public int Version { get; set; }
    public string Author { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public string Type { get; set; }
    public DateTimeOffset LastModified { get; set; }
    public string Status { get; set; }
    public string Visibility { get; set; }
}

File: server/worker/src/view-models/library-entry.view-model.ts
export interface LibraryEntryViewModel {
  ownerId: string;
  ownerName: string;
  entryId: string;
  version: number;
  author: string;
  title: string;
  description?: string;
  type: string;
  lastModified: Date;
  status: string;
  visibility: string;
}
